koku/koku/test_trino_db_utils.py

```diff
@@ -24,70 +30,95 @@ def test_executescript(self):
         Test execution of a buffer containing multiple statements
         """
         sqlscript = """
-drop table if exists hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}};
-create table hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}}
-(
-    id varchar,
-    i_data integer,
-    t_data varchar
-);
+            drop table if exists hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}};
+            create table hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}}
+            (
+                id varchar,
+                i_data integer,
+                t_data varchar
+            );
 
-insert into hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}} (id, i_data, t_data)
-values (cast(uuid() as varchar), 10, 'default');
+            insert into hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}} (id, i_data, t_data)
+            values (cast(uuid() as varchar), 10, 'default');
 
-insert into hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}} (id, i_data, t_data)
-values (cast(uuid() as varchar), {{int_data}}, {{txt_data}});
+            insert into hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}} (id, i_data, t_data)
+            values (cast(uuid() as varchar), {{int_data}}, {{txt_data}});
 
-select t_data from hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}} where i_data = {{int_data}};
+            select t_data from hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}} where i_data = {{int_data}};
 
-drop table if exists hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}};
-"""
+            drop table if exists hive.{{schema | sqlsafe}}.__test_{{uuid | sqlsafe}};
+        """
```

Comment:
Please don't change this. It is formatted this way because it is copy/pasted sql. It's ugly, sure, but it is easier to read/modify the sql when it's like this.

Reply:
It's the exact same SQL just indented. It can still be copy/pasted just as easily and makes the application code much more readable.

[Hiding whitespace](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#diff-view-options) makes changes like this easier to review (apologies if you already knew about that feature).